librz/debug/p/debug_native.c

```diff
@@ -418,7 +418,7 @@ static RzDebugReasonType rz_debug_native_wait(RzDebug *dbg, int pid) {
 }
 // FIXME: Should WAIT_ON_ALL_CHILDREN be a compilation flag instead of runtime debug config?
 #elif __linux__ && !defined(WAIT_ON_ALL_CHILDREN) // __WINDOWS__
-static RzDebugReasonType rz_debug_native_wait(RzDebug *dbg, int pid) {
+static int rz_debug_native_wait(RzDebug *dbg, int pid) {
```

Comment:
Linux native debug returns `RzDebugReasonType` as well and this is the signature in rz_debug:

```RZ_API RzDebugReasonType rz_debug_wait(RzDebug *dbg, RzBreakpointItem **bp);```

I am not sure if this should just be changed to a less indicative `int` fix a windows specific warning. I would prefer to separate this warning from the build issue.